doc/source/amqp1d.rst

```diff
@@ -0,0 +1,37 @@
+===================
```

Comment:
There is a list of flavor in the doc that needs to be updated too.

Reply:
done